packages/automl/frontend/src/app/components/common/FileExplorer/FileExplorer.tsx

```
@@ -0,0 +1,125 @@
+  name: string;
+  size: string;
+  path: string;
+}
+export type Files = File[];
+
+// Globals -------------------------------------------------------------------->
+
+// Components ----------------------------------------------------------------->
+
+// eslint-disable-next-line @typescript-eslint/no-empty-object-type
+interface SourceSelectorProps {}
```

Biome (2.4.4)

[error] An empty interface is equivalent to {}.

(lint/suspicious/noEmptyInterface)